src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -61,7 +61,60 @@ const en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }
 });
 
 // @TODO translations
-const cy: typeof en = en;
+const cy: typeof en = ({ isDivorce, isApplicant2, userCase, telephoneNumber, openingTimes }: CommonContent) => ({
+  applicationDownload: {
+    reference: 'Divorce-Application',
+    link: `/downloads/${isDivorce ? 'divorce-application' : 'application-to-end-civil-partnership'}`,
+    text: `Gweld y cais ${isDivorce ? 'am ysgariad' : 'application to end your civil partnership'} (PDF)`,
+  },
+  certificateOfServiceDownload: {
+    reference: 'Certificate-of-Service',
+    link: '/downloads/certificate-of-service',
+    text: "View your 'certificate of service' (PDF)",
+  },
+  respondentAnswersDownload: {
+    reference: 'Respondent-Answers',
+    link: '/downloads/respondent-answers',
+    text: `Gweld yr ymateb i'r cais ${isDivorce ? 'am ysgariad' : 'application to end your civil partnership'} (PDF)`,
```

Comment:
if you search for 'application to end your civil partnership' you can find it's translation on other pages